ci/travis-install.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 if [[ "${BUILDMODE}" == "ASTROPY" ]]; then
-    git clone --depth 1 git://github.com/astropy/ci-helpers.git
+#    git clone --depth 1 git://github.com/astropy/ci-helpers.git
+    git clone --depth 1 -b all-the-fixes git://github.com/djhoese/ci-helpers.git
```

Comment:
Can this be reverted now ?

Reply:
Yes. I'm guessing I'll still have issues though with the Windows conda activation.

Reply:
This is done but Windows still hates me.